### DISCUSSION TITLE
[Feature] create crowdin translation

### DISCUSSION THREAD

**Discussion by PterX** [immich-app/immich#7982] [Feature Request]
### The feature

please create an crowdin translation project,we can do it together

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
We've been intending to move to weblate, and have an account with them set up already, we just need to move the project over from the current platform.

**Reply by comradekingu** [reply]
Can I help with that?
If on Hosted Weblate, my account is https://hosted.weblate.org/user/kingu/ :)
